girder_worker/docker/transforms/__init__.py

```diff
@@ -0,0 +1,278 @@
+import sys
+import uuid
+import os
+import tempfile
+import six
+import abc
+
+from girder_worker_utils.transform import Transform
+
+
+TEMP_VOLUME_MOUNT_PREFIX = '/mnt/girder_worker'
+
+
+def _maybe_transform(obj, *args, **kwargs):
+    if hasattr(obj, 'transform') and hasattr(obj.transform, '__call__'):
+        return obj.transform(*args, **kwargs)
+
+    return obj
+
+
+class HostStdOut(Transform):
+    def transform(self, **kwargs):
+        from girder_worker.docker.io import (
+            StdStreamWriter
+        )
```

Comment:
Can these imports be done at the top of the file?

Reply:
Nope, these will only be available when ```transform``` is called on the Girder Worker side.